onnx_tf/handlers/backend/non_zero.py

```diff
@@ -0,0 +1,22 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("NonZero")
+class NonZero(BackendHandler):
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+    inp = kwargs["tensor_dict"][node.inputs[0]]
```

Comment:
I think it's probably better to use the full name `input` here to avoid confusion. But I'm fine with `inp` if you insist.

Reply:
@tjingrant sure, I will change it to "input"